### ISSUE TITLE
podman command in readme.md not showing stats

### ISSUE THREAD

**Issue by rcarrata** [RedHat-EMEA-SSA-Team/hetzner-ocp4#156]
In README.md for showing the stats of the Haproxy / LoadBalancer during the installation of the cluster, the command of podman is not working:

```
# podman exec -ti openshift-4-loadbalancer-demo ./watch-stats.sh
Error: no container with name or ID openshift-4-loadbalancer-demo found: no such container
```

A working alternative:
```
# LB_ID=$(podman ps | grep loadbalancer | awk '{ print $1 }')
# podman exec -ti $LB_ID ./watch-stats.sh
 # pxname               svname     status  last_chk
 ingress-http           FRONTEND   OPEN
```

**Comment by rbo** [comment]
Right, the container name ist `openshift-4-loadbalancer-${cluster_name}` maybe we should use the dummy variable.

**Comment by rbo** [comment]
Fixed in devel branch, will be merged into master via PR #159
